src/text/masked.rs

```diff
@@ -46,14 +46,14 @@ impl<'a> Masked<'a> {
 impl fmt::Debug for Masked<'_> {
     /// Debug representation of a masked string is the underlying string
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str(&self.inner)
+        fmt::Display::fmt(&self.inner, f)
```

Comment:
What makes this better / necessary? It seems more complex for no obvious reason (and below).

Reply:
This is a relic from before #1087 was created / merged. Basically: passing the object to its inner Debug/Display implementation.

`format!("{:>5}", "bla")` will be rendered as `"  bla"` when the padding stuff is correctly used. `write_str` ignores all this stuff and just renders `"bla"`. `f.pad(&self.inner)` would take into account more of this but it's simpler to just use the implementation of the inner one to have exactly the features it has.

Reply:
I think this probably needs tests so someone doesn't simplify it. It's also Worth pulling out of this PR, which is only refactoring changes, into another one that does actual changes.

Reply:
Removed from this PR